app/packs/controllers/datatable_approvals_controller.js

```diff
@@ -0,0 +1,14 @@
+import { Controller } from "stimulus"
+import { DataTable } from "simple-datatables"
+
+export default class extends Controller {
+  connect() {
+    new DataTable("#approvalsTable", {
```

Comment:
we can avoid the extra lookup and need for an id by passing `this.element` rather than a string. I know the others don't do this, but I feel this would be less fragile.

Reply:
@jcoyne thanks! This is done. However, I'll not that I tried this for the other tables and it didn't work (they did not get rendered actually). I'll take another poke at refactoring them in a separate PR if we want parity there.